tests/system/action/motion/test_update_metadata.py

```diff
@@ -4,7 +4,7 @@
 class MotionUpdateMetadataActionTest(BaseActionTestCase):
     def test_update_correct(self) -> None:
         self.create_model("meeting/2538", {"name": "name_jkPIYjFz"})
-        self.create_model("motion/111", {"meeting_id": 2538})
+        self.create_model("motion/111", {"meeting_id": 2538, "referenced_in_motion_recommendation_extension_ids": []})
```

Comment:
This should not be necessary since `[] == None` for the relation handling. Does it have a specific reason you added it?

Reply:
No, I just want to make sure, that this is not the reason of an error.

Reply:
ok

Reply:
I have removed the `referenced_in_motion_recommendation_extension_ids` again.